host/crypto/openssl/key.h

```diff
@@ -88,4 +96,15 @@ oe_result_t oe_public_key_verify(
     size_t signature_size,
     uint64_t magic);
 
+oe_result_t oe_sign_by_engine(
+    const char* engine_id, // string name of the engine, ie "esrp"
```

Comment:
Also remove esrp and esrp-specific references from header file.

Comment is also incomplete on line 103

Reply:
removed function decl